file1.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
```

Comment:
![Codacy](https://app.staging.codacy.org/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function. (wrap-iife)](https://app.staging.codacy.org/app/codacytestuser/CheckPrCommentsPublicProjectStaging/pullRequest?prid=4192)